gem/templates/profiles/viewprofile.html

```diff
@@ -65,5 +65,11 @@
                 </form>
             </div>
         </div>
+        <hr>
+        <div class="gender-form">
+            <div class="block">
+                    <h5>{% trans "Your gender is:" %}{{user.gem_profile.gender}}</h5>
```

Comment:
Currently this isn't working and I'm not sure why. `user` prints the username and `user.profile` prints the profile object but `user.gem_profile` doesn't print anything.